app/config.js

```diff
@@ -23,7 +23,7 @@ window.OPENSHIFT_CONFIG = {
   },
   additionalServers: [{
     protocol: "https",
-    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io:6443",
+    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io",
```

Comment:
Yeah I'm undecided on that. I think I'd want to see the connection error to know something isn't working?